extensions/ql-vscode/src/view/language-filter/LanguageFilter.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from "react";
+import {
+  QueryLanguage,
+  getLanguageDisplayName,
+} from "../../common/query-language";
+import { VSCodeDropdown, VSCodeOption } from "@vscode/webview-ui-toolkit/react";
+import { styled } from "styled-components";
+
+const languages = Object.values(QueryLanguage);
+
+const Dropdown = styled(VSCodeDropdown)`
+  width: 100%;
+  z-index: 9999;
```

Comment:
- `width: 100%` just seemed the nicest, but no strong opinions 😄 
- I don't think this `z-index` does anything currently, so probably worth removing